.gitignore

```diff
@@ -32,3 +32,6 @@
 
 # Ignore application configuration
 /config/application.yml
+
+/public
+/Gemfile.lock
```

Comment:
Remove this line. Gemfile.lock should not be ignored. It's changes should be committed only if you add new gem in Gemfile or changes version.